test/libnxz.0.63.0.abi

```diff
@@ -0,0 +1,2548 @@
+    <elf-symbol name='nx_inflateResetKeep' version='LIBNXZ_0.61.0' is-default-version='yes' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='nx_inflateSetDictionary' version='LIBNXZ_0.61.0' is-default-version='yes' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='nx_inflateSyncPoint' version='LIBNXZ_0.63.0' is-default-version='yes' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='nx_uncompress2' version='LIBNXZ_0.61.0' is-default-version='yes' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='nx_uncompress' version='LIBNXZ_0.61.0' is-default-version='yes' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='uncompress' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='uncompress2' version='ZLIB_1.2.9' is-default-version='yes' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='zlibVersion' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+  </elf-function-symbols>
+  <abi-instr version='1.0' address-size='64' path='crc32_ppc.c' comp-dir-path='/home/mscastanho/src/power-gzip/lib' language='LANG_C11'>
+    <function-decl name='__crc32_vpmsum' filepath='/home/mscastanho/src/power-gzip/lib/crc32_ppc.c' line='45' column='1' visibility='default' binding='global' size-in-bits='64'>
+      <parameter type-id='type-id-1'/>
+      <parameter type-id='type-id-2'/>
+      <parameter type-id='type-id-3'/>
+      <return type-id='type-id-1'/>
+    </function-decl>
+    <pointer-type-def type-id='type-id-4' size-in-bits='64' id='type-id-2'/>
+    <type-decl name='unsigned int' size-in-bits='32' id='type-id-1'/>
+    <type-decl name='unsigned long int' size-in-bits='64' id='type-id-3'/>
+    <type-decl name='unsigned char' size-in-bits='8' id='type-id-4'/>
+  </abi-instr>
+  <abi-instr version='1.0' address-size='64' path='gzip_vas.c' comp-dir-path='/home/mscastanho/src/power-gzip/lib' language='LANG_C11'>
+    <typedef-decl name='__useconds_t' type-id='type-id-1' filepath='/usr/include/bits/types.h' line='161' column='1' id='type-id-5'/>
+    <var-decl name='nx_fault_storage_address' type-id='type-id-6' visibility='default' filepath='/home/mscastanho/src/power-gzip/lib/nx_zlib.h' line='521' column='1'/>
+    <var-decl name='tb_freq' type-id='type-id-7' visibility='default' filepath='../inc_nx/nxu.h' line='71' column='1'/>
+    <function-decl name='usleep' filepath='/usr/include/unistd.h' line='469' column='1' visibility='default' binding='global' size-in-bits='64'>
```

Comment:
This is a system function in a system file.
Can we avoid tracking this?

Reply:
Done.